src/app/components/Notes/Modal/Modal.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from 'react'
+import { ButtonsContainer, ColorContainer, ColorDateContainer, HeadingContainer, InputSectionContainer, MainContainer, NameContainer, DateContainer, Buttons, Label } from './ModalStyle'
+import { ModelOverlay } from '../NotesStyle';
+
+interface ModelProps{
```

Comment:
create an other file with name types.ts in same folder and move all interfaces and types there